test/spec/widget.analog-scale.spec.js

```diff
@@ -10,3 +10,56 @@ const FORM2 = FORM1.replace( 'step="1"', 'step="0.1"' );
 
 runAllCommonWidgetTests( AnalogScaleWidget, FORM1, '2' );
 runAllCommonWidgetTests( AnalogScaleWidget, FORM2, '2.1' );
+
+const FORM_SHOW_SCALE =
+    `<label class="question non-select or-appearance-analog-scale or-appearance-vertical or-appearance-show-scale">
+        <span lang="" class="question-label active">Analog Scale</span>
+        <input type="number" name="/widgets/analog_scale/widget1" data-type-xml="int">
+    </label>`;
+
+testBasicInstantiation( AnalogScaleWidget, FORM_SHOW_SCALE );
+
+describe( 'Analog-scale widget with show scale', () => {
+
+    const fragment = document.createRange().createContextualFragment( FORM_SHOW_SCALE );
+    new AnalogScaleWidget( fragment.querySelector( 'input' ) );
+    const widget = fragment.querySelector( '.range-widget' );
+    const widgetInput = widget.querySelector( 'input' );
+    const widgetScalesContainer = widget.querySelector( '.range-widget__scale' );
```

Comment:
Interesting. So we don't need `beforeEach` hook because there is no change in the widget we are testing. Additionally, it made the test run longer because we have to set up these variables every time the test (`it`) runs.

Reply:
Got it. We use `const` because the variables will not be changed in these tests.